### PR TITLE
Fix ansible script for distributed environment

### PR DESCRIPTION
**Pull request by devbv**
Here's some minor changes for deploying ow in distributed environment.
Without these changes, you may fail in deploying with 'environment/distributed' in latest version.

### CODE REVIEW

File: ansible/roles/cli/tasks/deploy.yml [hunk 1]
```diff
@@ -24,8 +24,4 @@
   copy: src={{ openwhisk_cli.local.location }}/content.json dest={{ cli.nginxdir }}/
   when: cli_installation_mode == "local"
 
-- name: "copy the binaries from a local directory to Nginx directory"
```
Comment (rabbah) [inline ansible/roles/cli/tasks/deploy.yml]: why remove this?
@houshengbo @csantanapr can you comment on this?

Comment (devbv) [inline ansible/roles/cli/tasks/deploy.yml]: Exactly same with previous task (copy content.json)

**Comment by rabbah** [comment]
pg2/1919 🚀

**Comment by rabbah** [comment]
@devbv do you mind rebasing this to pick up the swift3 related fix? 
i can't run this in jenkins otherwise.

**Comment by rabbah** [comment]
Also do you have an Apache CLA on file?

**Comment by devbv** [comment]
@rabbah I just rebased it, Hope this works.
And I also have an Apache CLA file. Do you need it?

File: ansible/roles/cli/tasks/deploy.yml (hunk 1 above)
Comment (houshengbo) [inline ansible/roles/cli/tasks/deploy.yml]: I agree. It is a duplicate line of script. We can remove it.

**Pull request merged**